docpages/disdppgloss.md

```diff
@@ -0,0 +1,52 @@
+## A Glossary of Common Discord Terms
+
+Here is a list of common terms that are both terms one should know if one is to use D++ (or any other discord library) however are terms that are not commonly used throughout the discord community. This list, with a few exceptions, ***is discord specific***, that is to say that this is not a explanation of commonly used C++ terms, it is for people who are not familiar with the terminology of the discord API and libraries themselves.

```

Comment:
Can you reword the first sentence? Maybe split it into a two.
Also, the terms listed *are* commonly used throughout the discord (developer) community.